Day-II-Calculator/calculator/src/components/DisplayComponents/CalculatorDisplay.js

```diff
@@ -1,2 +1,11 @@
 import React from 'react';
 import './Display.css';
+
+const CalculatorDisplay = (props) => {
+    this.total = props.total;
```

Comment:
You can just display the props.total directly, it doesn't need to be stored in its own variable.